#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_drop-unused-code.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: pilotbmp.c, which doesn't work with VDR >= 1.5, isn't used in the
## plug-in, so just drop it.

@DPATCH@
diff -urNad vdr-plugin-pilot-0.0.9~/Makefile vdr-plugin-pilot-0.0.9/Makefile
--- vdr-plugin-pilot-0.0.9~/Makefile	2006-05-04 21:58:23.000000000 +0200
+++ vdr-plugin-pilot-0.0.9/Makefile	2008-02-10 14:49:05.000000000 +0100
@@ -45,7 +45,7 @@
 
 ### The object files (add further files here):
 
-OBJS = $(PLUGIN).o pilotosd.o pilotbmp.o pilotsetup.o 
+OBJS = $(PLUGIN).o pilotosd.o pilotsetup.o 
 
 ### Implicit rules:
 
diff -urNad vdr-plugin-pilot-0.0.9~/pilotosd.h vdr-plugin-pilot-0.0.9/pilotosd.h
--- vdr-plugin-pilot-0.0.9~/pilotosd.h	2006-05-04 21:58:23.000000000 +0200
+++ vdr-plugin-pilot-0.0.9/pilotosd.h	2008-02-10 14:49:19.000000000 +0100
@@ -4,7 +4,6 @@
 #include <vdr/osd.h>
 #include <vdr/menuitems.h>
 #include <ctype.h>
-#include "pilotbmp.h"
 
 static const cFont *font = cFont::GetFont(fontOsd);
 static const int LINEHEIGHT = font->Height();
